### PR TITLE
new navbar

### PR DESCRIPTION
**Pull request by XianBarbera**

### CODE REVIEW

**Review by ClaudioIrianni — APPROVED** [review]
nice navbar

File: src/App.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@ import Button from './components/Button.vue'
   <Navbar />
   <div class="content">
     <Card
-      title="Perro de mierda"
+      animal="Perro de mierda"
```
Comment (PawFV) [inline src/App.vue:12]: The name of the props begins to make little sense to me,
we have a `Card`, now if I want to use it for something other than animals, I have to pass `animal prop` which doesn't make sense.

Let's always think how we can name and build components in a way that can be reusable, not only for once specific thing.

I think we can rename this `prop` to `cardTitle` for now.

**Review by PawFV — CHANGES_REQUESTED** [review]

**Review by PawFV — CHANGES_REQUESTED** [review]

File: src/App.vue [hunk 2]
```diff
@@ -9,11 +9,11 @@ import Button from './components/Button.vue'
   <Navbar />
   <div class="content">
     <Card
-      title="Perro de mierda"
+      animal="Chiwawa, Angry Dog"
```
Comment (PawFV) [inline src/App.vue:12]: The name  `animal` for this `prop` make little sense.
We have a `Card`, now if I want to use it for something other than animals suppose, a list of T-shirts in the shopping section of the website, I have to pass `animal prop` which doesn't make sense.

Let's always think how we can name and build components in a way that can be reusable, not only for once specific thing.

I think we can rename this `prop` to `cardTitle` for now.

**Comment by XianBarbera** [comment]
how do you like the changes?

**Comment by PawFV** [comment]
> how do you like the changes?

@XianBarbera there are conflicts that block the merge to master, you need to solve them locally and push the changes.

**Review by PawFV — APPROVED** [review]

**Pull request merged**